context/ChatContext.tsx

```
@@ -0,0 +1,194 @@
+
+    return messages[conversationId] || [];
+}
+
+/**
+ * Hook to get unread conversations count
+ * Note: This is a simplified version - you may want to implement proper unread tracking
+ */
+export function useUnreadCount(): number {
+    const {isAuthenticated, user} = useAuth();
+    if (!isAuthenticated || !user) return 0;
+    const {conversations} = useChat();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)